certification.md

```diff
@@ -1,62 +1,49 @@
+    * [Confirm Booking](channel-manager-api.md#confirm-booking) 
+    * [Change Notification](channel-manager-api.md#change-notification) \(optional\)
+3. Email the endpoints to integrations@mewssystems.com.
+4. Set up the connection in the channel manager system.
+5. Use the [Get Properties](mews-api.md#get-properties) call to receive the `connectionToken` for the [Test Hotel](mews-api.md#test-environment).
+6. Pull hotel information, space mapping and rate mapping information using the [Get Configuration](mews-api.md#get-configuration) call.
+7. Map the [Test Hotel](mews-api.md#test-environment) in the UI of the channel manager backend using the data received in the [Get Configuration](mews-api.md#get-configuration) call.
+
+**Note:** Multi-property Mews customers (vacation rentals, apartments, villas, etc.) are often set up in Mews as spaces of one Mews property.  E.g. Property is the Mews Customer and Spaces are the rental properties.  These spaces (properties) can be sent to the channel manager as multiple connections, 1 connection token per space (property) or as a single connection for the entire property manager.
+
+## Certification Tests
+
+These tests will be done during a 60-minute call with the Mews Integrations team and the channel manager.  
+
+
+| Test | Requirement | Scenario | Expected result | Notes |
+| --- | --- | --- | --- | --- |
+| **Receive multi-space, multi-rate availability push** | Required | 5+ Spaces (3 Rooms, 1 Dorm and 1 Bed) and 2 Rates are mapped from the [Test Hotel](mews-api.md#test-environment) and pushed to the [Update Availability](channel-manager-api.md#update-availability) endpoint. | The channel manager returns ```{ "success": true}``` and data is updated in the channel manager extranet. | Mapping bed and dorm space types is recommended, not required. |
+| **Receive multi-space, multi-rate rate push** | Required | 5+ Spaces (3 Rooms, 1 Dorm and 1 Bed) and 2 Rates are mapped and pushed the [Update Prices](channel-manager-api.md#update-prices) endpoint. | The channel manager returns ```{ "success": true}``` and data is updated in the channel manager extranet. | Mapping bed and dorm space types is recommended, not required. |
+| **Receive multi-space, multi-rate restriction push** | Required | 5+ Spaces (3 Rooms, 1 Dorm and 1 Bed) and 2 Rates are mapped and pushed to the [Update Restrictions](channel-manager-api.md#update-restrictions) endpoint. | The channel manager returns ```{ "success": true}``` and data is updated in the channel manager extranet. | Mapping bed and dorm space types is recommended, not required. |
+| **Unmapped Space code error** | Required | An availability push with an unmapped Space type code is sent from the the [Test Hotel](mews-api.md#test-environment) to the [Update Availability](channel-manager-api.md#update-availability) endpoint. | The channel manager returns an error in [plain response](general-remarks.md#plain-response) that includes the unmapped Space type code. |     |
```

Comment:
... channel manager returns an error e.g. `{ "success": false, "error": { "code": XXX, "message": "e.g. Unknown space type mapping code 'ABC'." }}` as per [plain response](general-remarks.md#plain-response) ... 

IN note I'd mention that the mapping code shoudl be mentioned in the error message

Please fill in proper code.

Reply:
Or maybe the json coudl be in the separate column? would it look better? 

I only guess based on the "code"